### DISCUSSION TITLE
Webpage won't show properly

### DISCUSSION THREAD

**Discussion by dcosmember0** [community/community#118814] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to link a website I made in VScode to github, but when i added the files to the respiratory, the webpage would only show the readme file and not the webpages. I could use some help here!

**Comment by joemar25** [comment, marked as answer]
Examine your repository's file structure first things first. Your HTML, CSS, and JavaScript files should be in the root directory or in a public or docs folder, according to GitHub Pages. GitHub Pages may not be able to locate them if they are not in the proper location.
Next, confirm that the right branch is being used and that GitHub Pages is enabled for your repository. This setting is located in the "Pages" section of the repository settings. Occasionally, simply flipping a switch or selecting the appropriate branch can have a profound impact.
After you've adjusted those settings, commit and send your modifications to GitHub. Please be patient as it may take a minute or two to deploy. After that, check to see if your website appears as it should by refreshing your GitHub Pages URL.
Oh, and remember to verify the URL that you are using to visit your website one more time. An example of how it should appear is https://<username>.github.io/<repository>.
Check your workflow files if you're using GitHub Actions to make sure nothing is interfering with the deployment process.
